packages/core/src/ceramic.ts

```diff
@@ -682,7 +682,7 @@ export class Ceramic implements CeramicApi {
    * @param queries - Array of MultiQueries
    * @param timeout - Timeout in milliseconds
    */
-  async multiQuery(queries: Array<MultiQuery>, timeout?: number): Promise<Record<string, Stream>> {
+  async multiQuery(queries: Array<MultiQuery>, timeout = 7000): Promise<Record<string, Stream>> {
```

Comment:
Nice to have: add default value to the doc comment

Reply:
the autogenerated docs already include default values